language/move-lang/tests/move_check/expansion/public_main.exp

```diff
@@ -1,8 +1,8 @@
 error: 
 
-   ┌── tests/move_check/expansion/public_main.move:1:1 ───
+   ┌── tests/move_check/expansion/public_main.move:2:12 ───
    │
- 1 │ public fun main() {
-   │ ^^^^^^ Extraneous 'public' modifier. This top-level function is assumed to be public
+ 2 │ public fun main() {
+   │            ^^^^ Invalid script function. Scripts have a single function to serve as their entry point. It is thus uncessary to mark it 'public'
```

Comment:
I tried changing it because I don't plan on having them labeled public in the future. But I'll use your suggestion and we will change it again when/if the `transaction` keyword comes along